### PR TITLE
Fixed lower cases

### PR DESCRIPTION
**Pull request by emiliodeg**
Hello!

* Type: bug fix
* Link to issue: #14196 

**In raising this pull request, I confirm the following:**

- [x] I have read and understood the [Contributing Guidelines](https://github.com/phalcon/cphalcon/blob/master/CONTRIBUTING.md)
- [x] I have checked that another pull request for this purpose does not exist
- [ ] I wrote some tests for this PR
- [ ] I have updated the relevant CHANGELOG
- [ ] I have created a PR for the [documentation](https://github.com/phalcon/docs) about this change

Small description of change:
Just fixed the bug
Thanks

### CODE REVIEW

File: phalcon/Validation/Validator/File.zep [hunk 1]
```diff
@@ -102,7 +102,7 @@ class File extends ValidatorComposite
         // create individual validators
         for key, value in options {
             // min filesize
-            if strtolower(key) === "minSize" {
+            if strtolower(key === "minsize") {
```
Comment (ekmst) [inline phalcon/Validation/Validator/File.zep]: incorrect parentheses

File: phalcon/Validation/Validator/File.zep [hunk 2]
```diff
@@ -102,7 +102,7 @@ class File extends ValidatorComposite
         // create individual validators
         for key, value in options {
             // min filesize
-            if strtolower(key) === "minSize" {
+            if strtolower(key) === "minsize") {
```
Comment (ekmst) [inline phalcon/Validation/Validator/File.zep]: incorrect syntax :)

Comment (emiliodeg) [reply phalcon/Validation/Validator/File.zep]: yes I saw it after pushed it 🤦‍♂

**Comment by ekmst** [comment]
@emiliodeg 
may it makes sense to use the strcasecmp()?
What do you think?

**Comment by emiliodeg** [comment]
oh yes. You're right that is a better solution. I'll change it now

**Comment by niden** [comment]
Thank you @emiliodeg

**Review by niden — APPROVED** [review]

**Pull request merged**